pages/services/edge-lb/1.2/usage/index.md

```diff
@@ -47,10 +49,11 @@ The properties of this reload enable strategies like
 
 A change to the load balancer pool (such as adding a secret) will trigger a relaunch of all load balancers in the pool. This relaunch has the following properties:
 
-- Traffic is dropped. To minimize the impact, we suggest running more than one load balancer within the pool.
+- Traffic currently flowing through the given load balancer is dropped when it is stopped. To minimize the impact, we suggest running more than one load balancer within the pool.
+- Only one load balancer is stopped in the pool during the update at a time.
```

Comment:
I think the `at at time` should be closer to stopped. What about.

- During pool updates, only one load balancer is stopped at a time.